packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- Added `Mutex` lock in the `updateNftMetadata` function ([#4325](https://github.com/MetaMask/core/pull/4325))
+- Token price API support for mantle network ([#4376](https://github.com/MetaMask/core/pull/4376))
+
+### Changed
+
+- **BREAKING:** `NftController` constructor argument `selectedAddress` has been removed. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:get{Account,SelectedAccount}` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:selectedEvmAccountChange` event. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftDetectionController` now requires `AccountsControlelr:getSelectedAccount` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `AccountTrackerController` now requires a controller messenger ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Removed `getIdentities`, `getSelectedAddress` and `onPreferencesStateChange` from the constructor arguments of the `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Bump dependency and peer dependency `@metamask/accounts-controller` to `^17.0.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+- Upgrade `TokenRatesController` to `BaseControllerV2` ([#4314](https://github.com/MetaMask/core/pull/4314))
+- Update Nft controllers to use `selectedAccountId` instead of `selectedAddress` ([#4221](https://github.com/MetaMask/core/pull/4221))
+- `TokenRatesController` uses checksum instead of lowercase format for token addresses ([#4377](https://github.com/MetaMask/core/pull/4377))
+- Bump `@metamask/keyring-api` to `^8.0.0` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/eth-snap-keyring` to `^4.3.1` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/keyring-controller` to `^17.1.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+
+### Removed
+
+- Remove `value` from price API responses ([#4364](https://github.com/MetaMask/core/pull/4364))
```

Comment:
- Recategorized entries.
- We're moving away from "controller-messenger" in favor of "messenger" (https://github.com/MetaMask/decisions/blob/main/decisions/core/0001-messaging-non-controllers.md#decision-outcome).
- Additions to action/event allowlists are not breaking changes.
  - "Requires" isn't the best description for the functionality of action/event allowlists. We have previously used "allows" for actions and "subscribes to" for events.
- Constructor "option" instead of "argument", to avoid the impression that users can directly pass in constructor options object properties as function arguments.
- Added entry on new export.

```suggestion
### Added

- **BREAKING:** Add `messenger` as a constructor option for `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
(https://github.com/MetaMask/core/pull/4225))
  - Add `AccountTrackerControllerMessenger` type
- `NftController` now allows `AccountsController:getAccount`, `AccountsController:getSelectedAccount` as messenger actions and subscribes to the `AccountsController:selectedEvmAccountChange` messenger event ([#4221](https://github.com/MetaMask/core/pull/4221))
- `NftDetectionController` now allows `AccountsController:getSelectedAccount` as a messenger action ([#4221](https://github.com/MetaMask/core/pull/4221))
- Add `Mutex` lock in the `updateNftMetadata` function ([#4325](https://github.com/MetaMask/core/pull/4325))
- Token price API support for mantle network ([#4376](https://github.com/MetaMask/core/pull/4376))

### Changed

- **BREAKING:** Bump dependency and peer dependency `@metamask/accounts-controller` to `^17.0.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
- Upgrade `TokenRatesController` to `BaseControllerV2` ([#4314](https://github.com/MetaMask/core/pull/4314))
- Update Nft controllers to use `selectedAccountId` instead of `selectedAddress` ([#4221](https://github.com/MetaMask/core/pull/4221))
- `TokenRatesController` uses checksum instead of lowercase format for token addresses ([#4377](https://github.com/MetaMask/core/pull/4377))
- Bump `@metamask/keyring-api` to `^8.0.0` ([#4405](https://github.com/MetaMask/core/pull/4405))
- Bump `@metamask/eth-snap-keyring` to `^4.3.1` ([#4405](https://github.com/MetaMask/core/pull/4405))
- Bump `@metamask/keyring-controller` to `^17.1.0` ([#4413](https://github.com/MetaMask/core/pull/4413))

### Removed

- **BREAKING:** Remove `NftController` constructor options `selectedAddress`. ([#4221](https://github.com/MetaMask/core/pull/4221))
- **BREAKING:** Remove `AccountTrackerController` constructor options `getIdentities`, `getSelectedAddress` and `onPreferencesStateChange`  ([#4225]
- Remove `value` from price API responses ([#4364](https://github.com/MetaMask/core/pull/4364))
```